python/llm/src/ipex_llm/transformers/npu_pipeline_model/convert_pipeline.py

```diff
@@ -189,6 +189,8 @@ def convert_llm(model: torch.nn.Module,
                 max_prompt_len: int,
                 transpose_value_cache: bool,
                 group_size: int):
+    # whether to set layernorm weight as const
+    layernorm_const = os.environ.get("IPEX_LLM_LAYERNORM_CONST", "1") == "1"
```

Comment:
I feel it would be better to make it as a input parameter instead of env var ?

Reply:
@jason-dai Not sure how you do think of this ?

Reply:
Which method will set the parameter then? And how does the user specify it?

Reply:
I would suggest each model has a default value for this, which is based on our test result with better performance? Normally users won't know this value.

Reply:
For now, binbin use `set IPEX_LLM_LAYERNORM_CONST` to  control it.
If we change to input var, we can update this parameter in 
`model = AutoModelForCausalLM.from_pretrained(model_path, const_layernorm=True, ...`, and it's dafault value can be None.